### DISCUSSION TITLE
Domain verification

### DISCUSSION THREAD

**Discussion by teco-mishima** [WalletConnect/walletconnect-monorepo#3482] [Developer Support] [answered]
I am using Wallet Connect in a project under development.
I have a few questions about domain verification.

Is domain verification required to use Wallet connect in a production environment?
If not, are there any concerns?

Also, are there any concerns if verification has already been done on another project using the same domain?

**Comment by glitch-txs** [comment, marked as answer]
Hi, you don't need verification :)
